src/interfaces/PivotTable.ts

```diff
@@ -40,15 +40,33 @@ export enum UIClick {
     DOUBLE_CLICK = 2,
 }
 
+export interface IResizedColumnsItem {
+    width: number;
+    source: ColumnEventSourceType;
+}
+
 export interface IResizedColumns {
-    [columnIdentifier: string]: { width: number; source: ColumnEventSourceType };
+    [columnIdentifier: string]: IResizedColumnsItem;
+}
+
+export type ColumnWidthItem =
+    | IAttributeColumnWidthItem
+    | IMeasureColumnWidthItem
+    | IAllMeasureColumnWidthItem;
+export type ColumnWidthValue = number;
```

Comment:
maybe `AbsoluteColumnWidth`, `FixedColumnWidth`, or something similar?

Reply:
this names are base FO and its structure validated by arch.

Reply:
I do not think our internal naming is something relevant for architects. I just think the `Value` suffix is really just an empty word and adds no value here. The `FixedColumnWidth` at least explain it's semantics.

Reply:
done